src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
```

Comment:
It's better to use more specific selectors to avoid potential issues with multiple tables on the page. Consider adding a class or id to the table and use it in the selector.